client/src/connection.ts

```diff
@@ -224,6 +224,7 @@ export class ConnectionManager implements IConnectionManager {
 
     if ((this.connectionConfiguration.host == undefined) || (this.connectionConfiguration.host == '')) {
       args.push('--ip=127.0.0.1');
+      args.push('--local-workspace=' + vscode.workspace.textDocuments[0]);
```

Comment:
Could not tell if this was the best way to do it. There's also [workspaceFolders](https://code.visualstudio.com/docs/extensionAPI/vscode-api#_workspace).